frontend/src/Store/States/Session.state.ts

```diff
@@ -1,14 +1,13 @@
-import { CardData } from "../../Interfaces/CardData";
+import { Session } from "../../Interfaces/SessionData";
+
+export interface SessionState extends Session {}
 
-export interface SessionState {
-  id: string;
-  name: string;
-  cardSet: any;
-  cards: Array<CardData>;
-}
 export const initialState: SessionState = {
   id: "",
   name: "",
   cardSet: {},
   cards: [],
+  icon: "",
+  created: "",
+  lastUpdate: "",
```

Comment:
This value isn't being set anywhere.
Please ensure this is getting set anytime a card is reviewed.